interfaces/ldap/v0/schema.py

```diff
@@ -0,0 +1,55 @@
+"""This file defines the schemas for the provider and requirer sides of this
+relation interface.
+
+It must expose two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+"""
+
+from pydantic import AnyUrl, BaseModel, Field
+
+from interface_tester.schema_base import DataBagSchema
+
+
+class LdapProviderData(BaseModel):
+    ldap_url: AnyUrl = Field(
+        description="The LDAP URL",
+        example="ldap://ldap.canonical.com",
+    )
+    base_dn: str = Field(
+        description="The base entry as the starting point for LDAP search "
+                    "operation",
+        example="dc=canonical,dc=com",
+    )
+    bind_dn: str = Field(
+        description="The DN of the bind account",
+        example="cn=admin,ou=engineering,dc=canonical,dc=com",
+    )
+    bind_password: str = Field(
```

Comment:
Feel free to keep it as it is, not sure if there is a convention for this.

Reply:
I see this conversation was resolved, but there is no context for the resolution. What is the rationale for keeping this field named like this instead of any of the alternatives suggested here?

Reply:
I did not see a unified naming convention for passing juju secrets in the repo. So after discussing with @nsklikas, we decided to follow the old way.

IMHO, attaching the suffix does not help make it clearer that this field contains juju secret id or url.

Reply:
`bind_password_secret` strikes me as a reasonable name.